### ISSUE TITLE
Used alongside standard Devise broke the Devise mail confirmation

### ISSUE THREAD

**Issue by scriptamanent** [lynndylanhurley/devise_token_auth#265]
I'm using devise_token_auth to authenticate with api.
I need only auth, not registration or confirmation: for the registration I'm using Devise on my website.

The gem broke my Devise confirmation module: "send_confirmation_notification?" override the default Devise method.

```
m=  User.new.method(:send_confirmation_notification?)
=> #<Method: User(DeviseTokenAuth::Concerns::User)#send_confirmation_notification?>
```

Devise confirmation mails are blocked by this gem...

**Comment by killerham** [comment]
I'm having the same issue

**Comment by fabianoarruda** [comment]
Having the same issue.

**Comment by ysyyork** [comment]
The same issue here #456

**Comment by vic700208** [comment]
I have the same issue

**Comment by zachfeldman** [comment]
Closed in favor of #456, which has a gist workaround.
